website/docs/plugins/signing.html.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "registry"
+page_title: "Plugin Signing"
+sidebar_current: "docs-plugins-signing"
+description: |-
+  Terraform plugin signing trust levels
+---
+
+# Plugin Signing
+
+There are four levels of signing and programattic authentication for plugins in Terraform:
+
+* **Signed by HashiCorp** - are built, signed, and supported by HashiCorp. Official provider plugins can 
+typically be used without specifying provider source information in your Terraform configuration.
+* **Signed by Trusted Partners** - are built, signed, and supported by a third party. HashiCorp has 
+verified the ownership of the private key and we provide a chain of trust to the CLI to verify this
+programatically. To use partner providers in your Terraform configuration, you need to specify the
+provider source, typically this is the namespace and name to download from the registry.
+* **Self-signed** - are built, signed, and supported by a third party. HashiCorp does not provide a verification or chain of trust for the signing. You will want to obtain and validate fingerprints manually if you want to ensure you are using a binary you can trust.
```

Comment:
> You will want to obtain and validate fingerprints manually if you want to ensure you are using a binary you can trust.

It's not clear to me how a user could take action on this recommendation, unless we give some more detail about how to obtain the key fingerprint, and how to usefully validate it. As I understand it that is the subject of some unspecified future work. Until we have something more concrete to offer here, I think it's better to omit this sentence.

Reply:
Until we offer an ability to authenticate, perhaps we should print a fingerprint for this self-signed category to the output for manual verification?

Reply:
@paultyng we need to limit scope creep, but we will definitely consider that suggestion for a future enhancement.  We don't have concrete plans yet, but a number of ideas to extend the community provider trust process in later releases.

Reply:
I added in this PR to output KeyID when its third party signed, maybe that will be sufficient?